### PR TITLE
User authentication and aligning page layouts

### PR DESCRIPTION
**Pull request by gamSeier**
## Description
1. Updated user model to allow for login authentication. Django will now track what user is logged in, which will allow us to update pages accordingly.
2. Made login page style same as registration page.
3. Added user image in the top right of homepage when user is logged in which has account and logout buttons.

## How did you test the changes?
Created a new account, logged into account. Confirmed homepage tracks the authentication and changes buttons in the top right corner. Logout button removes the user from the instance and returns to the original homepage. Migrations will need to be made to account for the change to the user model.

## Testing Screenshots (Optional)
Screenshot 2024-03-06 at 7 06 31 PM
Screenshot 2024-03-06 at 7 05 59 PM
Screenshot 2024-03-06 at 7 06 06 PM

### CODE REVIEW

File: healthConfig/settings.py [hunk 1]
```diff
@@ -83,14 +83,17 @@
...
         "ENGINE": "django.db.backends.mysql",
         "NAME": "HealthScore",
-        "USER": os.environ["DB_USER"],
-        "PASSWORD": os.environ["DB_PASSWORD"],
-        "HOST": os.environ["DB_HOST"],
+        "USER": "root",
+        "PASSWORD": "",  # change it with your sql server password for local testing
+        "HOST": "localhost",
+        # "USER": os.environ["DB_USER"],
+        # "PASSWORD": os.environ["DB_PASSWORD"],
```
Comment (shreyas2499) [inline healthConfig/settings.py]: Could you revert these changes?

File: healthScore/templates/login.html [hunk 2]
```diff
@@ -25,8 +108,16 @@ <h2>Login</h2>
                 <label for="password">Password:</label>
                 <input type="password" class="form-control" id="password" name="password" placeholder="Password">
             </div>
+            <div style="text-align:left; margin-top:20px;">
```
Comment (shreyas2499) [inline healthScore/templates/login.html]: Could you please add the styles in a class?

File: healthConfig/settings.py (hunk 1 above)
Comment (gamSeier) [reply healthConfig/settings.py]: Yeah will do.

File: healthScore/templates/login.html (hunk 2 above)
Comment (gamSeier) [reply healthScore/templates/login.html]: Updated

File: healthConfig/settings.py (hunk 1 above)
Comment (gamSeier) [reply healthConfig/settings.py]: Updated

**Review by jazwu — APPROVED** [review]

**Review by shreyas2499 — APPROVED** [review]

**Comment by jazwu** [comment]
viewHealthHistoryTestCase failed that's why travis ci failed. Could you please comment viewHealthHistoryTestCase temporarily to make it pass and then merge the code first so that we could develop new features based on your work?
